src/lib/structures/ty.ml

```diff
@@ -514,19 +514,20 @@ let type_body name args = Decls.body name args
 
 
 (* smart constructors *)
-let tunit = Text ([],Hstring.make "unit")
+(* HACK: we should create a unique Dolmen identifier here. *)
```

Comment:
We should instead use `DStd.Ty.Const.unit`.

Reply:
I don't know `Dolmen` already contains a unit constant. I'll change it.